torchmetrics/functional/text/ter.py

```diff
@@ -557,7 +557,7 @@ def _ter_compute(total_num_edits: Tensor, total_ref_length: Tensor) -> Tensor:
     return _compute_ter_score_from_statistics(total_num_edits, total_ref_length)
 
 
-def ter(
+def translation_edit_rate(
```

Comment:
what about the deprecation path here as you did in #714 ?

Reply:
this one have not been released yet so no need for deprecation
https://github.com/PyTorchLightning/metrics/blob/master/CHANGELOG.md#added